### ISSUE TITLE
cobbler import --path=rsync:// fails because of wrong conversion to absolute path

### ISSUE THREAD

**Issue by neskweek** [cobbler/cobbler#1903]
Following #1767

I am currently using v2.8.2-1.el7 from EPEL and this is not solved:

```
cobbler import --name=Base --arch=x86_64 --path rsync://root@MIRROR:/var/www/html/Base/

Tue Jun  5 19:16:42 2018 - INFO | import_tree; ['/rsync:/root@MIRROR:/var/www/html/Base', 'Base', None, None, None]
Tue Jun  5 19:16:42 2018 - INFO | importing from a network location, running rsync to fetch the files first
Tue Jun  5 19:16:42 2018 - INFO | running: rsync -a  '/rsync:/root@MIRROR:/var/www/html/Base/' /var/www/cobbler/ks_mirror/Base-x86_64 --progress
Tue Jun  5 19:16:43 2018 - INFO | received on stdout: sending incremental file list

sent 12 bytes  received 12 bytes  48.00 bytes/sec
total size is 0  speedup is 0.00

Tue Jun  5 19:16:43 2018 - DEBUG | received on stderr: rsync: change_dir "/rsync:/root@10.9.70.12:/var/www/html/Base" failed: No such file or directory (2)
rsync error: some files/attrs were not transferred (see previous errors) (code 23) at main.c(1052) [sender=3.0.9]

Tue Jun  5 19:16:43 2018 - INFO | Exception occured: <class 'cobbler.cexceptions.CX'>
Tue Jun  5 19:16:43 2018 - INFO | Exception value: 'Command failed'
Tue Jun  5 19:16:43 2018 - INFO | Exception Info:
```

**Comment by excelero-joe** [comment]
Same issue, with the latest release from epel-release (CentOS 7):

# cobbler version
Cobbler 2.8.3
  source: ?, ?
  build time: Mon May 28 12:41:53 2018

Tue Jun 19 07:14:53 2018 - DEBUG | received on stderr: rsync: change_dir "/root/rsync:/vault.centos.org/7.4.1708/os/x86_64" failed: No such file or directory (2)

Looks like it is padding the path with /root/.

**Comment by teissler** [comment]
@ixs posted a fix for this long time ago.

https://github.com/cobbler/cobbler/issues/1613#issuecomment-269203145

**Comment by jmaas** [comment]
Fixed by implementing the suggestion from #1613 .
Please wait for Cobbler 2.8.5
Thanks for reporting!
